index.js

```diff
@@ -3,30 +3,69 @@
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @returns {boolean} Returns `true` if all values in the array are numbers. Otherwise, return `false`.
  */
-function isValid(rolls) {}
+function isValid(rolls) {
+
+  const result = []
+  for (let i = 0; i <= rolls.length - 1; i++) {
+    rolls.push(i);
+    return true;
+
+
+  }
```

Comment:
The instructions states that in order to return `true` the elements in the array must all be numbers, otherwise return false. This means that if the array has an object, or string, or another array in it it should return `false`.

How can you check the datatype of the elements?